### PR TITLE
add Appgroupname api and uniqueness app group name

### PR DESCRIPTION
**Pull request by fadlinurhasan**

### CODE REVIEW

File: spec/requests/api/v2/infrastructures_spec.rb [hunk 1]
```diff
@@ -10,75 +10,142 @@
...
-      get api_v2_profile_index_path,
-        params: { access_token: @access_token, limit: 20},
-        headers: headers
-      json_response = JSON.parse(response.body)
-      expect(json_response.length).to eq(12)
-    end
-  end
-
-  describe 'Profile by Cluster Name API' do
+  # describe 'List Profile' do
```
Comment (ekarisky) [inline spec/requests/api/v2/infrastructures_spec.rb:13]: why we don't remove this?

File: spec/requests/api/v2/infrastructures_spec.rb [hunk 2]
```diff
@@ -145,156 +211,154 @@
...
-          access_token: @access_token,
-          cluster_name: helm_infrastructure.cluster_name,
-          username: user_a.username,
-        }, headers: headers
-
-        expect(response.status).to eq 403
-      end
-    end
-  end
+  # describe 'Profile for Curator' do
```
Comment (ekarisky) [inline spec/requests/api/v2/infrastructures_spec.rb:214]: why we don't remove this?

File: spec/requests/api/v2/infrastructures_spec.rb (hunk 1 above)
Comment (fadlinurhasan) [reply spec/requests/api/v2/infrastructures_spec.rb]: sry my bad, that should not commented

File: spec/requests/api/v2/infrastructures_spec.rb (hunk 2 above)
Comment (fadlinurhasan) [reply spec/requests/api/v2/infrastructures_spec.rb]: sry my bad, that should not commented

**Review by ekarisky — APPROVED** [review]

**Pull request merged**